controller/settingcontroller.php

```diff
@@ -13,6 +13,7 @@
 namespace OCA\Music\Controller;
 
 use \OCP\AppFramework\Controller;
+use OCP\AppFramework\Http;
```

Comment:
Not needed. use statements are always absolute (and this is what PHPStorm added ;) I also tested this...but maybe I need to add some more tests.